setup.py

```diff
@@ -4,12 +4,12 @@
 required = [line for line in open('requirements/base.txt').read().split("\n")]
 
 setup(
-    name='withings',
+    name='pywithings',
     version='0.4.0',
     description="Library for the Withings API",
-    author='Maxime Bouroumeau-Fuseau',
-    author_email='maxime.bouroumeau@gmail.com',
-    url="https://github.com/maximebf/python-withings",
+    author='Maxime Bouroumeau-Fuseau, and ORCAS',
```

Comment:
@brad I think we can remove Maxime Bouroumeau-Fuseau here so that people don't try to contact him at developer@orcasinc.com